### ISSUE TITLE
Fix Your Errors Gisselle / Dio

### ISSUE THREAD

**Issue by egingric** [egingric/2016-Racing-Game#240]
Can't find file /GA228/03/dblanco/Textures/T_dblanco_CaveStructure_occlusion' while loading GA228/03/gtirse/assets/materials/rock1.uasset and rock2.uasset

Failed to load /GA228/03/dblanco/TexturesT_dblanco_CaveStructure_occlusion Referenced by MaterialExpressionTextureSampleParameter2D_9

![gterrors](https://cloud.githubusercontent.com/assets/3945065/14318714/33423536-fbdc-11e5-8e55-fb463e06bac5.PNG)

@scarrollrcad @diosmiodio

**Comment by diosmiodio** [comment]
Those textures are gone and aren't supposed to be referenced anymore. Those were from my test rock. I'll check on this as soon as I get out of class

**Comment by egingric** [comment]
Exactly. Remove them from where they are being referenced guys. Thanks!
